### PR TITLE
Mandd/abbr improv

### PR DESCRIPTION
**Pull request by mandd**
--------
Pull Request Description
--------
##### What are the significant changes in functionality due to this change request?

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual.
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Regression tests have to complete successfully.
- [x] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test.
- [x] 6. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.

### CODE REVIEW

File: src/utils/nlpUtils/OPLparser.py [hunk 1]
```diff
@@ -16,113 +16,205 @@
...
+    self.OPLtextParser()
+    self.OPLparser()
+
+    
+  def OPLentityParser(self):
+    '''
+    This method extracts all the form and function entities out of the OPL html file and it puts them in two separate lists
+    @in: filename, file name containing the OPL text
+    @out: objectList, list, list of form elements contained in the OPL file
+    @out: processList, list, list of function elements contained in the OPL file
```
Comment (wangcj05) [inline src/utils/nlpUtils/OPLparser.py]: Please update the docstring, it seems this function accept no input argument, and return no output.

File: src/utils/nlpUtils/OPLparser.py [hunk 2]
```diff
@@ -16,113 +16,205 @@
...
+          elif element['color'] == '#000078':
+            if elem.lower() not in self.processList:
+              self.processList.append(elem.lower())   
+
+
+  def OPLtextParser(self):
+    '''
+    This method extracts all the sentences out of the OPL html file and it puts them in a list
+    @in: filename, file name containing the OPL text
+    @out: objectList, sentences, list of sentenced contained in the OPL file
```
Comment (wangcj05) [inline src/utils/nlpUtils/OPLparser.py]: update the docstring here, no input and no output for this function

File: src/utils/nlpUtils/OPLparser.py [hunk 3]
```diff
@@ -16,113 +16,205 @@
...
+        self.sentences[index] = self.sentences[index].lstrip().lower()
+
+      self.sentences.remove('')
+
+
+  def OPLparser(self):
+    '''
+    This method translates all the sentences create a graph structure
+    @in: filename: file name containing the OPL text
+    @out: objectList: sentences, list of sentenced contained in the OPL file
```
Comment (wangcj05) [inline src/utils/nlpUtils/OPLparser.py]: please update the docstring for this function, no input and no output for this function

**Review by wangcj05 — CHANGES_REQUESTED** [review]
@mandd I have some general comments for you to consider, in addition, there are some trailing whitespace issue which caused the failure of pre-test.

File: src/utils/nlpUtils/OPLparser.py (hunk 1 above)
Comment (mandd) [reply src/utils/nlpUtils/OPLparser.py]: fixed

File: src/utils/nlpUtils/OPLparser.py (hunk 2 above)
Comment (mandd) [reply src/utils/nlpUtils/OPLparser.py]: fixed

File: src/utils/nlpUtils/OPLparser.py (hunk 3 above)
Comment (mandd) [reply src/utils/nlpUtils/OPLparser.py]: fixed

**Review by wangcj05 — APPROVED** [review]
PR looks good.

**Comment by wangcj05** [comment]
PR looks good.

**Pull request merged**